katas/content/multi_qubit_gates/preparing_bell_state/Verification.qs

```diff
@@ -0,0 +1,19 @@
+namespace Kata {
```

Comment:
And VerifyExercise is in Kata namespace?
(Not Quantum.Kata.Reference for example. Just want to make sure what to use from now on).

Reply:
I'll rename `VerifyExercise` to `CheckSolution` under the `Kata.Verification` namespace. So, the fully qualified name will now be `Kata.Verification.CheckSolution`.

Reply:
Updated to use `Kata.Verification.CheckSolution`.